galite-tests/src/test/kotlin/org/kopi/galite/demo/client/ClientForm.kt

```diff
@@ -138,6 +169,10 @@ class Clients : FormBlock(6, 6, "Clients") {
     help = "The client zip code"
     columns(u.zipCodeClt)
   }
+
+  init {
+    nameClt[0] = "test"
+  }
 }
 
 fun main() {
```

Comment:
I.add `access { }` to : 
1.  **Block** -> _fireActiveRecordChanged()_
2. **DBlock** -> refresh ()_ :` access { fireActiveRecordChanged(model.activeRecord)  }`
3. **DForm** -> _blockRecordChanged()_ : `access {  content.setPosition(current, count)  }`

II. then under **_DGridBlock_** -> _updateEditors()_ : remove `rowController.editorField.reset()`